pdfdetach/lib/pdfdetach/main.rb

```diff
@@ -45,12 +47,12 @@ def saveall(opts = {})
   end
 
   def run(args)
-    output = if @binary_path.empty?
-      `LD_LIBRARY_PATH=#{@base_path}/lib #{@base_path}/pdfdetach #{args}`
-    else
-      `#{@binary_path} #{args}`
-    end
+    output = if @binary_path&.empty?
```

Comment:
the `&` what means? 🤔

Reply:
Optional chain. It like the `?.` in other languages